packages/core/src/components/forms/controls.tsx

```diff
@@ -129,13 +131,43 @@ const Control: React.SFC<IControlInternalProps> = ({
 // Switch
 //
 
-export interface ISwitchProps extends IControlProps {}
+export interface ISwitchProps extends IControlProps {
+    /**
+     * String to display within the switch control component when the switch is active ("on").
+     */
+    internalTextActive?: string;
```

Comment:
`innerLabelChecked`, defaults to `innerLabel` if omitted

i'm not 💯 on the name yet